pkg/ksym/ksym.go

```diff
@@ -140,66 +150,64 @@ func (c *cache) Resolve(addrs map[uint64]struct{}) (map[uint64]string, error) {
 	return res, nil
 }
 
-func unsafeString(b []byte) string {
```

Comment:
Perhaps I am missing something, but I think this wasn't needed

Reply:
cc @brancz

Reply:
premature optimization, only do this if we have measurable impact